tests/Integration/Database/SchemaBuilderTest.php

```diff
@@ -65,4 +65,26 @@ public function testRegisterCustomDoctrineType()
         $this->assertArrayHasKey(TinyInteger::NAME, Type::getTypesMap());
         $this->assertSame('tinyinteger', Schema::getColumnType('test', 'test_column'));
     }
+
+    public function testRegisterCustomDoctrineTypeASecondTime()
+    {
+        if ($this->driver !== 'sqlite') {
+            $this->markTestSkipped('Test requires a SQLite connection.');
+        }
+
+        Schema::registerCustomDoctrineType(TinyInteger::class, TinyInteger::NAME, 'TINYINT');
+
+        Schema::create('test', function (Blueprint $table) {
+            $table->string('test_column');
+        });
+
+        $blueprint = new Blueprint('test', function (Blueprint $table) {
+            $table->tinyInteger('test_column')->change();
+        });
+
+        $blueprint->build($this->getConnection(), new SQLiteGrammar);
+
+        $this->assertArrayHasKey(TinyInteger::NAME, Type::getTypesMap());
+        $this->assertSame('tinyinteger', Schema::getColumnType('test', 'test_column'));
+    }
```

Comment:
This test is identical to the one above it, but prior to this PR it failed, because registering custom Doctrine types only worked _once_ during test runs (the very first time).